featuretools/tests/conftest.py

```diff
@@ -152,6 +152,64 @@ def ks_diamond_es(pd_diamond_es):
     return ft.EntitySet(id=pd_diamond_es.id, entities=entities, relationships=relationships)
 
 
+@pytest.fixture(params=['pd_transactions_defaultvalue_es', 'dask_transactions_defaultvalue_es', 'ks_transactions_defaultvalue_es'])
+def transactions_defaultvalue_es(request):
```

Comment:
thoughts on shortening this to `default_value_es` and adjusting the other fixtures to match?

Reply:
Yeah, that would be cleaner for sure.

Reply:
@rwedge Updated